.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,20 @@
+name: Run unit tests
+
+# https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#on
+on:
+  pull_request:
+
+  push:
+    branches:
+      - develop
+
+jobs:
+  unit-tests:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Run unit tests
+        working-directory: ./docker/
+        run: make unit-tests
```

Comment:
Add Unit Test Reporter (copied this from my own GHA branch)

```suggestion
        run: make unit-tests
        
# https://github.com/EnricoMi/publish-unit-test-result-action
      - name: Publish Test Results
        uses: docker://ghcr.io/enricomi/publish-unit-test-result-action:v1
        if: always()
        with:
          github_token: ${{ github.token }}
          files: docker/PhpUnitTests.xml
```

Reply:
seems harmless enough...if this ends up breaking any builds, i.e., false-negative, it won't be worth keeping it.